### PR TITLE
feat: db boot sync

### PR DESCRIPTION
**Pull request by edisontim**
Naive solution to sync events db to torii on boot of lore-machine. If the db is empty or has a few elements, only the missing ones will be added

Closes https://github.com/The-Overlore/lore-machine/issues/24 and closes https://github.com/The-Overlore/lore-machine/issues/21

### CODE REVIEW

File: overlore/townhall/service.py [hunk 1]
```diff
@@ -36,33 +27,38 @@ async def service(websocket: WebSocketServerProtocol, extra_argument: dict[str,
...
+    open_ai_api_key = ""
+    open_ai_api_key = "OpenAI API Key" if OPENAI_API_KEY is None else OPENAI_API_KEY
     if TORII_WS is None:
         raise RuntimeError("Failure to provide WS url")
-
-    events_db = EventsDatabase.instance().init()
-    VectorDatabase.instance().init()
-    OpenAIHandler.instance().init(OPENAI_API_KEY)
+    if TORII_GRAPHQL is None:
+        raise RuntimeError("Failure to provide WS url")
```
Comment (cwastche) [inline overlore/townhall/service.py:41]: L39 and L41 have the same error message, might be wise to differentiate them

File: overlore/utils.py [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+import os
 from argparse import Namespace
 from typing import Any
 
 import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# TODO: have a way to differentiate dev/prod config
```
Comment (cwastche) [inline overlore/utils.py:12]: Create issue for this

**Comment by edisontim** [comment]
Closes #40

**Review by cwastche — APPROVED** [review]
Love the new config handler

**Review by jobez — APPROVED** [review]
Approved. But glancing at this makes me want to make sure the system fails as fast as possible when environment variables aren't set. Do we already handle this?

**Pull request merged**